be/src/agent/utils.cpp

```diff
@@ -91,23 +86,23 @@ AgentStatus MasterServerClient::finish_task(
             client_status = client.reopen(config::thrift_rpc_timeout_ms);
 
             if (!client_status.ok()) {
-                OLAP_LOG_WARNING("master client, get client from cache failed."
-                                 "host: %s, port: %d, code: %d",
-                                 _master_info.network_address.hostname.c_str(),
-                                 _master_info.network_address.port,
-                                 client_status.code());
+                OLAP_LOG_WARNING(
+                        "master client, get client from cache failed."
+                        "host: %s, port: %d, code: %d",
+                        _master_info.network_address.hostname.c_str(),
+                        _master_info.network_address.port, client_status.code());
```

Comment:
it seems that it your tool, Indentation can changes(line_70 VS line_89) . Some are aligned with the first parameter of the previous line after a newline, and some are left with 8 spaces. Did you do this intentionally?

Reply:
No. This is caused by `AlwaysBreakBeforeMultilineStrings` which is true in google's style. We can set it to false to allow the above code.